src/main/java/net/openhft/chronicle/network/TcpEventHandler.java

```diff
@@ -480,6 +481,8 @@ private void handleIOE(@NotNull final IOException e) {
             } else if (message != null && message.startsWith("An existing connection was forcibly closed")) {
                 Jvm.debug().on(getClass(), message);
             } else if (!(e instanceof ClosedByInterruptException)) {
+                Language.warnOnce();
```

Comment:
I like this very much. Is it worth considering centralising this into Nick's SocketExceptionUtil so we have a one-stop-shop for exception message-related stuff?